pkg/maputil/maputil.go

```diff
@@ -0,0 +1,41 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package maputil
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// MapBoolValueAsOpt will parse key as a command-line option.
+// If only key is specified will be treated as true,
```

Comment:
Why true? Default is false in most cases. Can we handle on the caller side.

Reply:
In case of `--security-opt xxx` should be equal with `--security-opt xxx=true`.